akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -206,8 +199,11 @@ private[akka] trait InboundContext {
  */
 private[akka] object AssociationState {
   def apply(): AssociationState =
-    new AssociationState(incarnation = 1, uniqueRemoteAddressPromise = Promise(),
-      quarantined = ImmutableLongMap.empty[QuarantinedTimestamp])
+    new AssociationState(
+      incarnation = 1,
+      uniqueRemoteAddressPromise = Promise(),
+      quarantined = ImmutableLongMap.empty[QuarantinedTimestamp],
+      outboundCompression = NoOutboundCompression)
```

Comment:
keeping outbound compression here makes sense, since it is per incarnation, and new incarnation means new AssociationState.